<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>AnalyzerPlugin::SpecifiedFunctions</class>
 <widget class="QDialog" name="AnalyzerPlugin::SpecifiedFunctions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specified Functions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Specified Functions</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filter">
     <property name="placeholderText">
      <string>Filter</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="function_list">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AnalyzerPlugin::SpecifiedFunctions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AnalyzerPlugin::SpecifiedFunctions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
